sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java

```diff
@@ -774,4 +775,24 @@ public void deleteContact(ContactIndexDto contact, Runnable callback) {
 				callback.run();
 			});
 	}
+
+	public VerticalLayout getContactViewTitleLayout(ContactDto contact) {
+		VerticalLayout titleLayout = new VerticalLayout();
+		titleLayout.addStyleNames(CssStyles.LAYOUT_MINIMAL, CssStyles.VSPACE_4, CssStyles.VSPACE_TOP_4);
+		titleLayout.setSpacing(false);
+
+		Label diseaseLabel = new Label(DiseaseHelper.toString(contact.getDisease(), contact.getDiseaseDetails()));
+		CssStyles.style(diseaseLabel, CssStyles.H3, CssStyles.VSPACE_NONE, CssStyles.VSPACE_TOP_NONE);
+		titleLayout.addComponents(diseaseLabel);
+
+		Label classificationLabel = new Label(contact.getContactClassification().toShortString());
```

Comment:
Use toString instead of toShortString since we have the space for the longer version